scripts/bigdl-submit

```diff
@@ -2,21 +2,64 @@
 
 set -e
 
-#detect paths
+# usage
+usage() { echo "Usage: $0 [-a|--archives <string#string> -h|--help]" 1>&2; exit 1; }
```

Comment:
will this --archives conflict with spark-submit's archive?

Reply:
No, I have store the original spark arguments in `cmd`.